src/wallet/asyncrpcoperation_sendmany.cpp

```diff
@@ -134,7 +139,7 @@ void AsyncRPCOperation_sendmany::main() {
         set_state(OperationStatus::FAILED);
     }
 
-    std::string s = strprintf("async rpc %s finished (status=%s", getId(), getStateAsString());
+    std::string s = strprintf("%s: finished (status=%s", getId(), getStateAsString());
     if (success) {
         s += strprintf(", tx=%s)\n", tx_.ToString());
```

Comment:
Request for Change: Here's where we want to ensure the full txid is logged. We could either do it here or have a 'verbose mode' of `ToString` or an alternate method that includes the full txid.